### DISCUSSION TITLE
References for feature addition

### DISCUSSION THREAD

**Discussion by hsbasu** [amit9838/mousam#86] [Ideas]
1. [They](https://github.com/linuxmint/cinnamon-spices-applets/tree/master/weather%40mockturtl) provides features like:
    1. Option to choose from multiple weather services.
    2. Choose units independently for temperature, pressure, wind speed etc.
    3. Auto refresh forecast at every user-defined time interval.

**Comment by amit9838** [comment]
Yes, I'm ok with option 3, but other two options will make the app overwhelming.

**Reply by hsbasu** [reply]
I agree with you, but option two is also feasible in the near future.

**Reply by amit9838** [reply]
Yes option 2 is doable. but lets keep it for the future

**Comment by hsbasu** [comment]
Some beautiful design references for different cards taken from [MSN Weather](https://www.msn.com/en-in/weather/):
1. current Weather card
![current_weather_style](https://github.com/amit9838/mousam/assets/41947504/bfa68083-93f5-4244-a1c1-9e7e7b43249f)
2. Today's suggestion card:
![suggestion_today](https://github.com/amit9838/mousam/assets/41947504/2002103c-0f5c-477b-bb85-0bf08bd107fb)
3. Sun/Moon Card:
![sun_moon_card](https://github.com/amit9838/mousam/assets/41947504/f3e8fca4-4b55-4b11-a92d-838ff4e6dc64)

**Reply by amit9838** [reply]
I think total daylight is useful information here
May be we can add moon info in the future

**Reply by hsbasu** [reply]
What about the current condition card?

**Reply by amit9838** [reply]
Yes we can improve current condition section, may be add more info,
For the bottom section , it looks good but we already have laid down same information in card format so  can't use that 🙂.
what else do you think we can add in current section?
